cmd/replicate-status.go

```diff
@@ -157,21 +158,40 @@ func (s replicateStatusMessage) String() string {
 		var hdrStr, hdrDet string
 		hdrStr = ep
 		if hdrStr != "" {
-			hdrDet = arn
+			hdrDet = console.Colorize("Values", arn)
 		} else {
 			hdrStr = arn
 		}
 		r := console.Colorize(th, newPrettyTable(" | ",
-			Field{"Ep", 120},
+			Field{"Ep", 100},
 		).buildRow(fmt.Sprintf("%s %s", coloredDot, hdrStr)))
 		rows += r
 		rows += "\n"
 		if hdrDet != "" {
 			r = console.Colorize("THeader", newPrettyTable(" | ",
-				Field{"Arn", 120},
-			).buildRow("  "+hdrDet))
+				Field{"Arn", 100},
+			).buildRow("  "+"ARN: "+hdrDet))
 			rows += r
 			rows += "\n"
+			bwStat, ok := s.ReplicationStatus.Stats[arn]
+			if ok && bwStat.BandWidthLimitInBytesPerSecond > 0 {
+				limit := humanize.Bytes(uint64(bwStat.BandWidthLimitInBytesPerSecond) * 8)
```

Comment:
Why is there "x 8" here ?

Reply:
Because the data sent is in bits @vadmeste

Reply:
the server stores bandwidth limit in bytes/sec whereas the mc UI takes bandwidth-limit flag in bits/sec - hence the conversion while displaying `mc replicate status`

Reply:
> the server stores bandwidth limit in bytes/sec whereas the mc UI takes bandwidth-limit flag in bits/sec - hence the conversion while displaying mc replicate status

understood, but also you are passing bits to humanize.Bytes(), I don't think it is working correctly.

Reply:
updated PR to display as Bytes/sec instead